### PR TITLE
DOC-3774: Docker docs improvements

### PR DESCRIPTION
**Pull request by AaronForinton**
Graph of the page changes: 
Screenshot 2022-02-18 at 10 37 40

- Moving Cordform and Dockerform configuration options out of the non-plugin page.
- Having the 'Official Corda Docker Image' page as an intro/directory page rather than a list of docker use cases, and a separate page for deployments.
- 'Running nodes remotely' -> the last header of the existing running nodes page was about remote node running, the rest of the page was explicitly about doing it locally, so this has moved into its own page.

### CODE REVIEW

**Comment by NinaKantiR3** [comment]
We need to think about the hierarchy of where the "Official Docker Image" page is hosted I think under operations it's hidden as a user looking for the Docker docs it's not obvious. @benpesterr3 any thoughts

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+* Provides a validating Notary service.
+* Runs the `corda-finance` CorDapp.
+
+`PartyA` and `PartyB` nodes, each of which:
+
+* Does not provide any services.
+* Runs the `corda-finance` CorDapp.
+* Has an RPC (Remote Procedure Call) user (`user1`), which enables you to log in the node via RPC.
+
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:38]: Your bulleted lists might look a bit strange here. Party A / notary / etc should be bulleted and then the rest indented.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        h2Port   10012
+        cordapps = ["$corda_release_distribution:corda-finance:$corda_release_version"]
+        // Grants user1 the ability to start the MyFlow flow.
+        rpcUsers = [[ user: "user1", "password": "test", "permissions": ["StartFlow.net.corda.flows.MyFlow"]]]
+    }
+}
+```
+
+{{< note >}}
+Make sure to use Corda gradle plug-in version 5.0.10 or above.
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:99]: This is a requirement, not really a note. Fine if this is here as a reminder, but it should be laid out in the requirements, and made clear if it's important enough for a note later.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+{{< /note >}}
+
+The configuration values used in the example are described below.
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:102]: No directional text pls.

Comment (benpesterr3) [reply content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:102]: The very next line is a title with 'configuration' in it, so you can probably just delete this.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+
+### Required configuration
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:104]: Can this be active voice please. Configure or Configuring your node (or whatever is appropriate)

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+# Cordform plug-in
+
+Corda provides two `gradle` plug-ins called `Cordform` and `Dockerform`. They both allow you to run tasks that automatically generate and configure a local set of nodes for testing and demonstration purposes. This page contains information about the operation of the Cordform plug-in. Visit the [Dockerform](../../../../../en/platform/corda/4.8/open-source/generating-a-node-dockerform.md) page for Dockerform configuration options.
+
+* A `Cordform` task creates nodes in the `build/nodes` directory. The example `Cordform` task used in this document creates three nodes: `Notary`, `PartyA`, and `PartyB`, however you are free to spin up more nodes, specify what nodes you need on the network, change node names, and update node configurations.
+* `Cordform` tasks require you to deploy each Corda node and database separately.
+
+## Tasks using the Cordform plug-in
+
+Run this example task to create the following three nodes in the `build/nodes` directory:
```
[the remark this answers is not in the document]
Comment (benpesterr3) [reply content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:26]: And if you're note repeating yourself, then pls explain exactly what the example task is.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md (hunk 4 above)
Comment (benpesterr3) [reply content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:104]: Actually - reading down, I'm not sure this should be H3 (you can check it back yourself) but the way this is organised feels a bit muddled for the user. This whole bit is about configuration, so spell out what the configurations can be - as an overview, include the required (minimum?) and the optional. Then do the examples. 

Introduce
Overview of main actions
Guide

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+
+{{< note >}}
+Make sure to use Corda gradle plug-in version 5.0.10 or above.
+{{< /note >}}
+
+The configuration values used in the example are described below.
+
+### Required configuration
+
+* `name` &lt;string&gt; - use this configuration option to specify the legal identity name of the Corda node. For more information, see [myLegalName](corda-configuration-fields.html#mylegalname). For example:
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:106]: Please check the style guide to make sure you are using the M dash correctly here.

Comment (benpesterr3) [reply content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:106]: (and throughout)

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+```kotlin
+sshd {
+  port = 2222
+}
+```
+
+You can extend the `deployNodes` task with more `node {}` blocks to generate as many nodes as necessary for your application.
+
+{{< warning >}}
+When adding nodes, make sure that there are no port clashes!
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:166]: Absolutely no exclamation marks please.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+{{< /warning >}}
+
+To extend node configuration beyond the properties defined in the `deployNodes` task, use the `configFile` property with the file path (relative or absolute) set to an additional configuration file. This file should follow the standard [Node configuration](corda-configuration-file.md) format of `node.conf`. The properties set there will be appended to the generated node configuration.
+
+{{< note >}}
+If you add a property to the additional configuration file that has already been created by the `deployNodes` task, both properties will be present in generated node configuration.
+{{< /note >}}
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:173]: There are a lot of notes in this document that are actually information to document, and not notes to highlight. When you serve locally, you shouldn't see too many notes cropping up, and when you do, they should be there for a good reason. 

I'm not saying take this one out, but please check the number of them being used.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 9]
```diff
@@ -0,0 +1,303 @@
...
+
+Alternatively, you can also add the path to the additional configuration file while running the gradle task via the `-PconfigFile` command-line option. However, this will result in the same configuration file being applied to all nodes.
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:175]: Be careful with alternatives. 

'Alternatively, you can also' could be understood as cumulative (not 'instead of' but 'in addition to'). 

More importantly, if there is more than one way of doing something, then break it out into a list that makes clear one of the following methods must be followed. 

Pls remember these are instructions, too much prose can be hard to follow. It's great for just passing on information but it makes these steps hard to follow.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 10]
```diff
@@ -0,0 +1,303 @@
...
+
+Following on from the previous example, the `PartyB` node in the next example below has additional configuration options added from a file called `none-b.conf`:
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:177]: Please review this and see if you can lay it out logically, rather than talking through it in this way. It's nice to read like this, but making all this text relational in this way makes it hard to consume as a guide. The user must read it like an article, which is not possible when you are also leaving this screen to go to another screen and coming back for your next instruction.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 11]
```diff
@@ -0,0 +1,303 @@
...
+        name "O=PartyB,L=New York,C=US"
+        [...]
+        drivers = ext.drivers + ['lib/my_specific_jar.jar']
+    }
+}
+```
+
+### Package namespace ownership
+
+To configure [package namespace ownership](../../4.8/enterprise/node/deploy/env-dev.html#package-namespace-ownership), use the optional `networkParameterOverrides` and `packageOwnership` blocks, in a similar way to how the configuration file is used by the [Network Bootstrapper](network-bootstrapper.md) tool. For example:
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:209]: Can you treat this as a step that does not require reference to other steps?

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 12]
```diff
@@ -0,0 +1,303 @@
...
+    [...]
+}
+```
+
+### Sign CorDapp `.jar` files
+
+The default Cordform behaviour is to deploy CorDapp `.jar` files “as built”.
+
+* Prior to Corda 4.0, all CorDapp `.jar` files were unsigned.
+* As of Corda 4.0, CorDapp `.jar` files created by the gradle `cordapp` plug-in are signed by a Corda development certificate by default.
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:232]: pre Corda 4.0 is probably not useful to anyone using this documentation. 

Also - pls define what "as built'' means, and check the style guide for usage of ""

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 13]
```diff
@@ -0,0 +1,303 @@
...
+
+You can use the Cordform `signing` entry to override and customise the signing of CorDapp `.jar` files.
+Signing a CorDapp enables its contract classes to use signature constraints instead of other types of constraints, such as [Contract Constraints](api-contract-constraints.md).
+
+The signing task may use an external keystore, or create a new one.
+You can use the following parameters in the `signing` entry:
+
+
+* `enabled` - the control flag to enable the signing process. It is set to `false` by default. Set to `true` to enable signing.
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:241]: Check use of em dash here pls.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 14]
```diff
@@ -0,0 +1,303 @@
...
+  * `keypass` - the private key password, if it is different from the keystore password. The default value is `cordacadevkeypass`.
+  * `storetype` - the keystore type. The default value is `JKS`.
+  * `dname` - the distinguished name for the entity. Only use this option when `generateKeystore` is set to `true` (see below).
+  * `keyalg` - the method to use when generating a name-value pair. The default value is `RSA` because Corda does not support `DSA`. Only use this option when `generateKeystore` is set to `true` (see below).
+* `generateKeystore` - the flag to generate a keystore. The default value is `false`. If set to `true`, an "ad hoc" keystore is created and its key is used instead of the default Corda development key or any external key. The same `options` to specify an external keystore are used to define the newly created keystore. In addition,
+  `dname` and `keyalg` are required. Other options are described in [GenKey task](https://ant.apache.org/manual/Tasks/genkey.html). If the existing keystore is already present, the task will reuse it. However if the file is inside the `build` directory,
+  then it will be deleted when the gradle `clean` task is run.
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:253]: This is a fairly chunky bullet point, with an additional requirement. Consider how this is laid out, maybe add indented bullet.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md [hunk 15]
```diff
@@ -0,0 +1,303 @@
...
+
+  ```groovy
+  task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
+      includeWhitelist = [ "net.corda.finance.contracts.asset.Cash", "net.corda.finance.contracts.asset.CommercialPaper" ]
+      //...
+  ```
+
+### Optional migration step
+
+If you are migrating your database schema from an older Corda version to Corda 4.8, you must add the following parameter to the node section in the `build.gradle` and set it to `true`, as follows:
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:288]: 'As follows' is redundant here.

File: content/en/platform/corda/4.8/open-source/node-docker-deployments.md [hunk 16]
```diff
@@ -0,0 +1,194 @@
...
+    weight: 1150
+tags:
+- node
+- docker
+title: Docker deployments
+---
+
+# Docker deployments
+
+This page contains a repository of example manual node configuration operations using Docker. Before executing any of these commands,
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/node-docker-deployments.md:20]: Can you have a look at this intro and economise a bit? There are a few lines here - for more information on this... this page contains... where you could be more ruthless. 

This is a style issue, but it will help the document I think.

File: content/en/platform/corda/4.8/open-source/node-docker-deployments.md [hunk 17]
```diff
@@ -0,0 +1,194 @@
...
+ensure the Corda Docker image has been correctly configured. For more information on this, see the [Official Corda Docker Image](docker-image.md).
+
+The node runs in a container. This is similar to executing the jar directly, except with the need to mount directories into the container that a Corda node would normally have. It is also necessary to set up
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/node-docker-deployments.md:23]: Please review this document and make sure you are using `.jar` (not jar) throughout.

File: content/en/platform/corda/4.8/open-source/node-docker-deployments.md [hunk 18]
```diff
@@ -0,0 +1,194 @@
...
+* Config - the node config must be mounted at location `/etc/corda/node.config`
+* Logging - all log files will be written to location `/opt/corda/logs`
+
+If using the H2 database:
+
+
+* Persistence - the folder to hold the H2 database files must be mounted at location `/opt/corda/persistence`
+
+{{< note >}}
+If there is no dataSourceProperties key in the node.conf, the Docker container overrides the url for H2 to point to the persistence directory by default so that the database can be accessed outside the container
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/node-docker-deployments.md:62]: Check this for code  ` ` requirements.

File: content/en/platform/corda/4.8/open-source/node-docker-deployments.md [hunk 19]
```diff
@@ -0,0 +1,194 @@
...
+        -v /home/user/sharedFolder/node-infos:/opt/corda/additional-node-infos \
+        -v /home/user/sharedFolder/network-parameters:/opt/corda/network-parameters \
+        -p 10200:10200 \
+        -p 10201:10201 \
+        corda/corda-zulu-java1.8-4.8:latest
+```
+
+There is a new mount `/home/user/sharedFolder/node-infos:/opt/corda/additional-node-infos` which is used to hold the `nodeInfo` of all the nodes within the network.
+As the node within the container starts up, it will place its own nodeInfo into this directory. This will allow other nodes also using this folder to see this new node.
+
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/node-docker-deployments.md:92]: New when? Try to avoid 'new' as these docs should be around a long while, definitely outlasting any mount's newness.

**Review by benpesterr3 — COMMENTED** [review]
Hey Aaron - this is mostly reading well, but I think you need to go through it with the style guide at hand and give it a once over. Also I have left some notes to move it away from (nicely written) informational prose, into a (more boring) more efficient user guide. Sometimes this reads well, but would be difficult to use whilst performing tasks. This is mostly just about organising the information and being ruthless about lists, steps, and superfluous language.

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (AaronForinton) [reply content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:26]: Think two lines could be swapped around here to make it read more clearly, I've committed this

File: content/en/platform/corda/4.8/open-source/node-docker-deployments.md (hunk 19 above)
Comment (AaronForinton) [reply content/en/platform/corda/4.8/open-source/node-docker-deployments.md:92]: !! Thank you

**Comment by NinaKantiR3** [comment]
> > We need to think about the hierarchy of where the "Official Docker Image" page is hosted I think under operations it's hidden as a user looking for the Docker docs it's not obvious. @benpesterr3 any thoughts
> 
> @NinaKantiR3 - might need a bit of a conversation about this. The main menu for OS is divided into tasks. Operations is a high level menu item for node operators and network operators.
> 
> The alternative would be in Upgrading CorDapps and Nodes. Do you think this would be better? Can sat for sure that this by itself is not top level documentation, so needs to belong in one of the main categories listed there. Happy to talk more. TBH we anticipate most of the docs to be discovered via search.

Discusses with Ben on this lets keep it in operations and link to the docket "master" page in the getting started journey

File: content/en/platform/corda/4.8/open-source/node-docker-deployments.md [hunk 20]
```diff
@@ -0,0 +1,193 @@
...
+tags:
+- node
+- docker
+title: Docker deployments
+---
+
+# Docker deployments
+
+This page contains a repository of example manual node configuration operations using Docker. Before executing any of these commands,
+ensure the [Corda Docker image](docker-image.md) has been correctly configured.
```
Comment (ceebee-dev) [inline content/en/platform/corda/4.8/open-source/node-docker-deployments.md:21]: We don't talk about configuring a docker image.  We just want to make sure we use the correct docker command or docker-compose command.

Comment (AaronForinton) [reply content/en/platform/corda/4.8/open-source/node-docker-deployments.md:21]: I think removing 'configuration' clears things up a little in that case

File: content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md (hunk 11 above)
Comment (AaronForinton) [reply content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md:204]: I'm going to flag this as a separate documentation ticket because it's going to require a convo with an SME who is as-yet unidentified.

**Comment by AaronForinton** [comment]
@benpesterr3 you'll notice I left a note about one of your comments RE: content - I don't think that's going to be doable as part of this ticket but happy to raise the ticket on it, it's by itself a discrete piece of work that'll take half a day and this work has already been extended several times over the 3 story points I initially gave it!

File: content/en/platform/corda/4.8/enterprise/docker-image.md [hunk 21]
```diff
@@ -11,175 +11,23 @@ weight: 120
 ---
 
 
-# Official Corda Docker image
+# Official Corda Docker Image
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/docker-image.md:14]: I believe this is a lower case I on image. Sentence case for titles.

Comment (benpesterr3) [reply content/en/platform/corda/4.8/enterprise/docker-image.md:14]: If not then Docker image and Docker Image are used inconsistently here so pls check this.

File: content/en/platform/corda/4.8/enterprise/node-docker-deployments.md [hunk 22]
```diff
@@ -0,0 +1,188 @@
...
+the source CorDapps. In this example, there are two CorDapps provided `corda-insurance.jar` and `corda-kyc.jar`
+
+```shell
+docker run -ti \
+        -v /home/user/docker/docker/config:/etc/corda \
+        -v /home/user/docker/docker/certificates:/opt/corda/certificates \
+        -v /home/user/docker/docker/persistence:/opt/corda/persistence \
+        -v /home/user/docker/docker/logs:/opt/corda/logs \
+        -v /home/user/corda/samples/bank-of-corda-demo/build/nodes/BankOfCorda/cordapps:/opt/corda/cordapps \
+        entdocker.software.r3.com/corda-enterprise-java1.8-4.8:latest db-migrate-create-jars
```
Comment (ceebee-dev) [inline content/en/platform/corda/4.8/enterprise/node-docker-deployments.md:168]: The  image entdocker.software.r3.com/corda-enterprise-java1.8-4.8:latest no longer seems to exist.
However, corda/corda-enterprise-alpine-zulu-java1.8-4.8.6:latest, from docker hub does.

File: content/en/platform/corda/4.8/enterprise/node-docker-deployments.md [hunk 23]
```diff
@@ -0,0 +1,188 @@
...
+It is also possible to use the image to directly perform the migration of the database.
+
+```shell
+docker run -ti \
+        -v $(pwd)/docker/config:/etc/corda \
+        -v $(pwd)/docker/certificates:/opt/corda/certificates \
+        -v $(pwd)/docker/persistence:/opt/corda/persistence \
+        -v $(pwd)/docker/logs:/opt/corda/logs \
+        -v $(pwd)/samples/bank-of-corda-demo/build/nodes/BankOfCorda/cordapps:/opt/corda/cordapps \
+        entdocker.software.r3.com/corda-enterprise-java1.8-4.8:latest db-migrate-execute-migration
```
Comment (ceebee-dev) [inline content/en/platform/corda/4.8/enterprise/node-docker-deployments.md:185]: Image no longer seems to exist seem comment above.

File: content/en/platform/corda/4.8/open-source/node-docker-deployments.md [hunk 24]
```diff
@@ -0,0 +1,193 @@
...
+```shell
+docker run -ti \
+        -v /home/user/docker/docker/config:/etc/corda \
+        -v /home/user/docker/docker/certificates:/opt/corda/certificates \
+        -v /home/user/docker/docker/persistence:/opt/corda/persistence \
+        -v /home/user/docker/docker/logs:/opt/corda/logs \
+        -v /home/user/corda/samples/bank-of-corda-demo/build/nodes/BankOfCorda/cordapps:/opt/corda/cordapps \
+        entdocker.software.r3.com/corda-enterprise-java1.8-4.8:latest db-migrate-create-jars
+```
+
```
[the remark this answers is not in the document]
Comment (ceebee-dev) [reply content/en/platform/corda/4.8/open-source/node-docker-deployments.md:175]: ...and this is about using the enterprise version of corda in the open source docs.

File: content/en/platform/corda/4.8/enterprise/node-docker-deployments.md [hunk 25]
```diff
@@ -0,0 +1,188 @@
...
+## Joining an existing Compatibility Zone
+
+{{< note >}}
+Requirements: A Compatibility Zone, the Zone Trust Root and authorisation to join said Zone.
+
+{{< /note >}}
+It is possible to use the image to automate the process of joining an existing Zone as detailed [here](../../../../../en/platform/corda/4.8/enterprise/network/compatibility-zones.md)
+
+The first step is to obtain the Zone Trust Root, and place it within a directory. In the below example, the Trust Root is stored at `/home/user/docker/certificates/network-root-truststore.jks`.
+It is possible to configure the name of the Trust Root file by setting the `TRUST_STORE_NAME` environment variable in the container.
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/node-docker-deployments.md:101]:
```suggestion
You can configure the name of the Trust Root file by setting the `TRUST_STORE_NAME` environment variable in the container.
```

File: content/en/platform/corda/4.8/enterprise/node-docker-deployments.md [hunk 26]
```diff
@@ -0,0 +1,188 @@
...
+* `MY_PUBLIC_ADDRESS` - The public address to advertise the node on.
+* `NETWORKMAP_URL` - The address of the Zone’s network map service (this should be provided to you by the Zone).
+* `DOORMAN_URL` - The address of the Zone’s doorman service (this should be provided to you by the Zone).
+* `NETWORK_TRUST_PASSWORD` - The password to the Zone Trust Root (this should be provided to you by the Zone).
+* `MY_EMAIL_ADDRESS` - The email address to use when generating the config. This must be the same as registered with the Zone.
+* `SSHPORT` - The port to use for SSH.
+* `RPC_USER` - The RPC username.
+
+There are some optional variables which allow customisation of the generated config:
+
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/node-docker-deployments.md:131]: please can you squish these double line breaks.

File: content/en/platform/corda/4.8/enterprise/node-docker-deployments.md [hunk 27]
```diff
@@ -0,0 +1,188 @@
...
+
+* `MY_P2P_PORT` - The port to advertise the node on (defaults to 10200). If changed, ensure the container is launched with the correct published ports.
+* `MY_RPC_PORT` - The port to open for RPC connections to the node (defaults to 10201). If changed, ensure the container is launched with the correct published ports.
+
+Once the container has finished performing the initial registration, the node can be started as normal
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/node-docker-deployments.md:136]: This sentence has no ending.

File: content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-cordform.md [hunk 28]
```diff
@@ -0,0 +1,288 @@
...
+tags:
+- generating
+- node
+title: Cordform plug-in
+weight: 2
+---
+
+### Tasks using the Cordform plug-in
+
+Run this example task to create the following three nodes in the `build/nodes` directory:
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-cordform.md:16]: How do they run the example task? Change of language maybe.

File: content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-cordform.md [hunk 29]
```diff
@@ -0,0 +1,288 @@
...
+        cordapps = ["$corda_release_distribution:corda-finance:$corda_release_version"]
+        // Grants user1 the ability to start the MyFlow flow.
+        rpcUsers = [[ user: "user1", "password": "test", "permissions": ["StartFlow.net.corda.flows.MyFlow"]]]
+    }
+}
+```
+
+{{< note >}}
+Make sure to use Corda gradle plugin version 5.0.10 or above.
+{{< /note >}}
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-cordform.md:88]: This should only be a note if it is a reminder of minimum version requirements. If not, please create a requirements section and include this.

File: content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-cordform.md [hunk 30]
```diff
@@ -0,0 +1,288 @@
...
+    identifier: corda-enterprise-4-8-corda-nodes-deploying-node-generating-cordform
+    parent: corda-enterprise-4-8-corda-nodes-deploying-generate
+tags:
+- generating
+- node
+title: Cordform plug-in
+weight: 2
+---
+
+### Tasks using the Cordform plug-in
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-cordform.md:14]: Should not be h3. This is the first thing on the page. Can you fix this title pls.

File: content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-dockerform.md [hunk 31]
```diff
@@ -0,0 +1,346 @@
...
+| dbName                 | Database name                                     |
+| dbPort                 | Database port (default: 5432)                     |
+| dbHostName             | Database host (default: localhost)                |
+| dbInit                 | Initialisation script for Postgres Docker image   |
+| dbDockerfile           | Wrapper of base Postgres Docker image             |
+| dbDataVolume           | Path to database files for Postgres Docker image  |
+
+To make the database files persistent across multiple `docker-compose` runs, you must set the `dbDataVolume` parameter. If this variable is commented out, the database files will be removed after every `docker-compose` run.
+
+#### Run the Dockerform task
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/node/deploy/generating-a-node-dockerform.md:257]: These header values are a bit off. We should really only be using H4 for very detailed information. Can you take a look at rationalising these a bit.

File: content/en/platform/corda/4.8/enterprise/operations/deployment/generating-a-node-dockerform.md [hunk 32]
```diff
@@ -0,0 +1,346 @@
...
+    identifier: corda-enterprise-4-8-operations-guide-deployment-node-generating-dockerform
+    parent: corda-enterprise-4-8-operations-guide-deployment-node-generating
+tags:
+- generating
+- node
+title: Dockerform plug-in
+weight: 3
+---
+
+### Tasks using the Dockerform plug-in
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/operations/deployment/generating-a-node-dockerform.md:14]: Please fix the headers throughout this doc. Should never start with a H3.

File: content/en/platform/corda/4.8/enterprise/operations/deployment/generating-a-node-dockerform.md [hunk 33]
```diff
@@ -0,0 +1,346 @@
...
+{{< /note >}}
+
+2. To create the nodes defined in the `prepareDockerNodes` gradle task added in the first step, run the following command in a command prompt or a terminal window, from the root of the project where the `prepareDockerNodes` task is defined:
+
+* Linux/macOS: `./gradlew prepareDockerNodes`
+* Windows: `gradlew.bat prepareDockerNodes`
+
+This command creates the nodes in the `build/nodes` directory. A node directory is generated for each node defined in the `prepareDockerNodes` task. The task also creates a `docker-compose.yml` file in the `build/nodes` directory.
+
+{{< note >}}
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/operations/deployment/generating-a-node-dockerform.md:334]: This note is massive. Is it really necessary for it to be a note? Just give it a H2 and take away the note.

File: content/en/platform/corda/4.8/open-source/running-a-node-remotely.md [hunk 34]
```diff
@@ -0,0 +1,37 @@
...
+    weight: 1140
+tags:
+- running
+- node
+title: Running nodes remotely
+---
+
+# Running nodes remotely
+
+{{< note >}}
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/open-source/running-a-node-remotely.md:19]: Does this make sense? Is this whole doc just for testing and the real 'running a node remotely' doc is somewhere else? Can you figure out a way of making this make better sense?

**Review by benpesterr3 — COMMENTED** [review]
Most of my comments are almost definitely issues that were pre-existing in these docs, but while we have them open we should fix the worst of the inconsistencies. 

Mostly issues are to do with Header usage and some notes that really don't make sense.

**Review by benpesterr3 — APPROVED** [review]
